tests/coalaTest.py

```diff
@@ -322,3 +322,18 @@ def test_coala_without_color(self):
                 self.assertRegex(err, '^\[WARNING\]')
             self.assertEqual(
                 retval, 0, 'coala must return zero when there are no errors')
+
+    def test_coala_ignore_file(self):
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                    coala.main, 'coala',
+                    '-c', os.devnull,
+                    '--non-interactive',
+                    '-f', filename,
+                    '-i', filename,
```

Comment:
why `-i` and `--ignore` ?

Reply:
To test the ignore flag as defined in https://github.com/coala/coala/issues/4886#issue-275183478

Reply:
they are two flags which have the same impact, and it is bizarre to use both together.  Nobody would do this, and a test doing it proves nothing.

Reply:
Alright. I will write a different test for `--ignore` :sweat_smile:

Reply:
What part of coala are you testing better by having a test for `-i` and `--ignore`.
Which lines of code?

Reply:
This test checks the working of ignore flag which can be used as `-i` or `--ignore`, which is a cmdline option. So we need to test both of them.

The definition of this  cmdline option is stored in https://github.com/coala/coala/blob/4e0c88e5d5e575e6fd20b5abb437d0ef54cb6077/coalib/parsing/DefaultArgParser.py#L148-L150

Reply:
You are right. Irrespective of the `add_argument` same code will get executed.

Reply:
I think a single test should suffice the task.